doc/whats_new.rst

```diff
@@ -12,6 +12,51 @@ Version 0.18
 Changelog
 ---------
 
+.. _model_selection_changes:
+
+Model Selection Enhancements and API Changes
+--------------------------------------------
+
+  - **The ``model_selection`` module**
+
+    The classes and functions from the formerly :mod:`sklearn.cross_validation`,
+    :mod:`sklearn.grid_search` and :mod:`sklearn.learning_curve` are
+    grouped together into a single module available at
+    :mod:`sklearn.model_selection`. There are some major changes to these
+    newly moved classes and functions. Refer below to know about the
+    changes.
+
+  - **Data-independent CV splitters enabling nested cross-validation**
+
+    The new cross-validation splitters, defined in the
+    :mod:`sklearn.model_selection`, are not initialized with any
+    data-dependent parameters. Instead they expose a :func:`split` method
+    that takes in the data and yields a generator for the different splits.
+
+    This change makes it possible to do nested cross-validation with ease,
+    facilitated by :class:`model_selection.GridSearchCV` and similar
+    utilities.
+
+  - **The enhanced `results_` attribute**
+
+    The new ``results_`` attribute (of :class:`model_selection.GridSearchCV`
+    and similar utilities) introduced in lieu of the ``grid_scores_``
```

Comment:
> but are unlikely to have that all merged for 0.18

You mean to say the addition of other `SearchCV` classes or addition of the `results_` attribute to `EstimatorCV` classes?